docs/reference-guides/metric_update_procedure.md

```diff
@@ -1,30 +1,32 @@
 # Metric Update Procedure
 
-Please follow the steps below to update the following Google Sheets:
+Please follow the steps below to update the following Looker Dashboards and Google Sheets:
 
-- [Autopush-rs Metrics][1]
-- [FxA Metrics][2]
-- [Merino-py Metrics][3]
+- [Looker Dashboards][1]
+- [Autopush-rs Metrics][2]
+- [FxA Metrics][3]
+- [Merino-py Metrics][4]
```

Comment:
I use the numbers as a personal preference because they take up less room, but I concede that they are flawed. I changed the strategy across all docs (see https://github.com/mozilla/ecosystem-test-scripts/pull/37/commits/bbbdaa6b88bca7d4f3f7ea5b07882674d5289cc4)